client/db/test/dbtest.go

```diff
@@ -60,13 +60,7 @@ func RandomMatchProof(sparsity float64) *db.MatchProof {
 		proof.SecretHash = randBytes(32)
 	}
 	if !doZero() {
-		proof.SecretKey = randBytes(32)
-	}
-	if !doZero() {
-		proof.InitStamp = rand.Uint64()
-	}
-	if !doZero() {
-		proof.RedeemStamp = rand.Uint64()
```

Comment:
`RedeemStamp` and `InitStamp` were moved the the `MatchAuth` struct early in development. These are just artifacts.